regcore_pgsql/views.py

```diff
@@ -11,7 +12,7 @@ def matching_sections(search_args):
     sections_query = Document.objects\
         .annotate(rank=SearchRank(
             F('documentindex__search_vector'), SearchQuery(search_args.q)))\
-        .filter(rank__gte=0.15)\
+        .filter(rank__gt=settings.PG_SEARCH_RANK_CUTOFF)\
```

Comment:
Ah yes, I should have added that to the commit. The use case I was thinking of was returning all matches, regardless of particular ranking. Non-matches will have a rank of zero, so to achieve this goal we want to say match anything *strictly greater* than zero. We could simulate it with `>=` via 0.0000001 or what have you, but I figured this was a little cleaner.